geomesa-convert/geomesa-convert-shp/src/main/scala/org/locationtech/geomesa/convert/shp/ShapefileConverterFactory.scala

```diff
@@ -88,6 +88,9 @@ object ShapefileConverterFactory extends LazyLogging {
     } catch {
       case NonFatal(e) =>
         logger.debug(s"Could not infer Shapefile converter from path '$path':", e)
+        if (logger.underlying.isTraceEnabled()) {
+          e.printStackTrace()
```

Comment:
we shouldn't print to stdout - but isn't the stack trace already being logged on the line above?

Reply:
I think those logger calls drop out the first line of the stack trace rather than the whole thing...  I can live without this if need be.

Reply:
Cool, I'll 86 that line.